rls/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+load("//:java_grpc_library.bzl", "java_grpc_library")
+
+java_library(
+    name = "rls",
+    srcs = glob([
+        "src/main/java/io/grpc/rls/*.java",
+    ]),
+    visibility = ["//visibility:public"],
+    deps = [
+        ":rls_java_grpc",
+        "//api",
+        "//core",
+        "//core:internal",
+        "//core:util",
+        "//stub",
+        "@com_google_code_findbugs_jsr305//jar",
+        "@com_google_guava_guava//jar",
+        "@io_grpc_grpc_proto//:rls_java_proto",
```

Comment:
If other languages make a breaking change to grpc-proto and grpc-java will fail, we they notice in the first time?

Reply:
at least, they will notice when they are trying to import to google3. although there shouldn't be any breaking changes.